README.md

```diff
@@ -0,0 +1,27 @@
+# Cookes and Sessions
+
+## Objective
+
+We're going to make a very simple shopping cart, stored entirely in the session.
+
+## Introduction
+
+The Rails `session` method gives us access to the Rails session. The session is a datastore implemented with cookies. You can store simple data structures in the session. ActiveRecord models, no. Arrays of strings or numbers, yes. Basically, if it could be converted into JSON, it can go in the `session`.
+
+We're going to use this to implement a shopping site. Here's how the site will work:
+
+  1. The page has an input box on it.
+  2. The user types in the item they want and click `add to cart`.
+  3. The item is added to their cart. The page shows everything in the user's cart.
+
+## Instructions
+
+You'll find that the rspec is failing. Get them to pass. I would tackle it like this:
+
+1. `ApplicationController#cart`s tests are failing because, primarily, it doesn't exist. Write it to make them pass.
+2. Now `rails generate` a Products controller with two actions, index and add.
+3. Fill them in to make the specs pass.
+
+After the specs pass, you'll find you still don't have an app, for you have no views. Rails will tell you which ones you need to write.
```

Comment:
I would include some capybara tests to solve this issue. Somethign like this https://github.com/learn-co-curriculum/rails-pollywog-ranch/blob/5155f9d5137794692d36d50a188607f0c424e0c3/spec/03_features/metamorphosize_spec.rb#L29-L33